modules/custom/d_p_side_by_side/config/update/d_p_side_by_side.8121.yml

```diff
@@ -0,0 +1,63 @@
+core.entity_form_display.paragraph.d_p_side_by_side.default:
+  add:
+    content:
+      field_d_long_text:
+        type: text_textarea
+        weight: 1
+        label: hidden
+        region: content
+        settings:
+          rows: 5
+          placeholder: ''
+        third_party_settings: {  }
+      field_d_main_title:
+        type: string_textfield
+        weight: 0
+        label: hidden
+        region: content
+        settings:
+          size: 60
+          placeholder: ''
+        third_party_settings: {  }
+  change:
+    expected:
+      content:
+        field_d_settings:
+          weight: 15
+        field_d_cta_link:
+          weight: 16
+    new:
+      content:
+        field_d_settings:
+          weight: 16
+        field_d_cta_link:
+          weight: 15
+  delete_value:
+    - parents: hidden:config
+      values:
+        - field_d_long_text
+        - field_d_main_title
+core.entity_view_display.paragraph.d_p_side_by_side.default:
+  add:
+    content:
+      field_d_long_text:
+        type: text_textarea
+        weight: 1
+        region: content
+        settings:
+          rows: 5
+          placeholder: ''
+        third_party_settings: {  }
+      field_d_main_title:
+        type: string_textfield
+        weight: 0
+        region: content
+        settings:
+          size: 60
+          placeholder: ''
+        third_party_settings: {  }
+  delete_value:
+    - parents: hidden:config
+      values:
+        - field_d_long_text
+        - field_d_main_title
```

Comment:
ten plik wydaje się niepotrzebny po (moich) ostatnich zmianach, w ktorych robie praktycznie to samo 
![image](https://user-images.githubusercontent.com/37533838/88180292-6ea31e80-cc2d-11ea-9d3c-629fbd3a89a5.png)

hmm, w ogole mam wrazenie, ze po wykonaniu tego update po domergowaniu developa obecnego, w logach powinny byc jakies info ze update nie przeszly, bo juz w poprzednim kroku zmieniane sa jakies wagi (field_d_settings po tym bedzie mial wage 16 :thinking: )
![image](https://user-images.githubusercontent.com/37533838/88180431-aad67f00-cc2d-11ea-8d24-ea7d519dc9d0.png)